import.py

```diff
@@ -1,58 +1,68 @@
 import weaviate
+from weaviate import Config
+import weaviate.classes as wvc
 import json
-import helper
-
-client = weaviate.Client("http://localhost:9999")
-client.timeout_config = (10000) 
-
-client.schema.delete_all()
-schema = {
-    "classes": [
-        {
-            "class": "Podcast",
-            "properties": [
-                {
-                    "name": "title",
-                    "dataType": ["text"]
-                },
-                {
-                    "name": "transcript",
-                    "dataType": ["text"]
-                }
-            ]
-        }
-    ]
-}
-client.schema.create(schema)
+import os
+from langchain.text_splitter import RecursiveCharacterTextSplitter
 
-with open("data/podcast_ds.json", 'r', encoding="utf-8") as f:
-    datastore = json.load(f)
-    
-def add_podcasts(batch_size = 1):
-    no_items_in_batch = 0
-    for item in datastore:
-        podcast_object = {
-            "title": item["title"],
-            "transcript": item["transcript"]
-        }
+# Starting up the weaviate client
+client = weaviate.Client(
+    "http://localhost:8080",
+    additional_config=Config(grpc_port_experimental=50051),
+    additional_headers={
+        "X-OpenAI-Api-Key": os.environ["OPENAI_APIKEY"],  # Replace with your key
+    },
+)
 
-        podcast_uuid = helper.generate_uuid('podcast', item["title"] + item["url"])
-        client.batch.add_data_object(podcast_object, "Podcast", podcast_uuid)
-        no_items_in_batch += 1
+print(client.collection.delete("Podcast"))
 
-        if no_items_in_batch >= batch_size:
-            results = client.batch.create_objects()
+# Creating a new collection named Podcast
+client.collection.create(
+    name="Podcast",
+    properties=[
+        wvc.Property(
+            name="title",
+            data_type=wvc.DataType.TEXT,
+        ),
+        wvc.Property(
+            name="transcript",
+            data_type=wvc.DataType.TEXT,
+        )
+    ],
+    vectorizer_config=wvc.ConfigFactory.Vectorizer.text2vec_openai()
+)
 
-            for result in results:
-                    if result['result'] != {}:
-                        helper.log(result['result'])
+# Checking if the collection is created successfully
+print(client.collection.exists("Podcast"))
+
+with open("./data/podcast_ds.json", 'r') as f:
+    datastore = json.load(f)
 
-            message = str(item["title"]) + ' imported'
-            helper.log(message)
```

Comment:
I think it would be nice, if we could keep the logging function for useful console outputs

Reply:
Ok, I will do that.